packages/core/src/actors/observable.ts

```diff
@@ -76,6 +87,11 @@ export function fromObservable<T, TEvent extends EventObject>(
       };
     },
     start: (state, { self }) => {
+      if (state.status === 'done') {
+        // Do not restart a completed observable
```

Comment:
what about a canceled actor? a canceled actor is pretty much "completed" (even if not successfully). OTOH, a canceled actor might not get persisted at all, right? Since it should be cleaned up from `state.children`.

This also brings an important issue here - spawned actors (including stopped ones) might be part of the `context`. We are not doing anything about this and likely we should. They should be rehydrated in the same way as the invoked ones - the fact that they live in `context` complicates the matter though :s

Reply:
For spawned actors, follow-up PR?

Reply:
👍